Skills.md

```diff
@@ -13,6 +13,7 @@ This API currently supports versions 4.1 and 4.2. Version is passed in the Accep
 | language            | No | A string determining the language (total 22 languages supported) in which the input text is written. Default value is en. Note that the input parameter passing to language should be the language id. |
 | threshold      | No | A double value between 0 and 1 controlling minimum relevancy scores for skill recognition. Higher values will more tightly restrict the returned skill tags. Default is 0.5. A threshold of 0 means all seed skill phrases recognized by exact string matching will be returned. Note that this parameter is only supported for inputs in English. |
 | auto_thres     | No | A boolean value indicating whether automatic thresholding is desired. If enabled, then when input text contains 150 or fewer words, the threshold parameter will be overwritten to 0 and all confidence values will be overwritten to 1.0&#42;. Default is true.  Note that this parameter is only supported for inputs in en,fr, and de. |
+| return_relevant_skills | No | A boolean value indicating whether to return relevant skills for each extracted skill. Defaults to false. Relevant skills are identical to extracted skills, with the confidence rating their relevance to the extracted skill. |

```

Comment:
I thought about that Gavin, but I looked at the results and turns out the confidences tend to vary in tight groups (probably an artifact of the scoring?). So for example some skills have relevant skills all between 0.9 and 0.8, others all between 0.7 and 0.6. If we added a threshold, we'd get many skills where the threshold has no effect, and many skills with no relevant skills. If we need to threshold I think it makes more sense to take the top N.